test/test_timeframe.jl

```diff
@@ -243,86 +243,114 @@ end
       let tf = tf"1A"
         @test isdefined(tf, :period)
         @test d + tf == Date(2018, 12, 1)
+        @test tf + d == Date(2018, 12, 1)
         @inferred d + tf
+        @inferred tf + d
       end
 
       let tf = tf"1AS"
         @test isdefined(tf, :period)
         @test d + tf == Date(2018, 12, 1)
+        @test tf + d == Date(2018, 12, 1)
```

Comment:
[DRY!](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

why not defining 

    d_exp = Date(2018, 12, 1)

Reply:
I'm just lazily copy-paste it :p